### PR TITLE
Add Fuel 3.2 changelog, murano docs

### PR DESCRIPTION
**Pull request by mattymo**
Ready to merge

### CODE REVIEW

**Comment by mihgen** [comment]
please rebase

File: pages/frequently-asked-questions/0010-rabbitmq.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ RabbitMQ Cluster Restart Issues Following A Systemwide Power Failure
   simultaneously. RabbitMQ requires that after a full shutdown of the cluster,
```
Comment (mihgen) [inline pages/frequently-asked-questions/0010-rabbitmq.rst:6]: Please remove this net_verify_failure.jpg. Error message is confusing there, and I already asked Dmitry to fix it. Especially I don't want it be shown in docs in such unclear format.

**Comment by mattymo** [comment]
rebased

**Comment by mihgen** [comment]
+1

**Pull request merged**